tests/v2/validation/provisioning/rke2/etcd_node_delete_test.go

```diff
@@ -0,0 +1,127 @@
+package rke2
+
+import (
+	"net/url"
+	"testing"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	v1 "github.com/rancher/rancher/tests/framework/clients/rancher/v1"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters"
+	"github.com/rancher/rancher/tests/framework/extensions/machinepools"
+	nodestat "github.com/rancher/rancher/tests/framework/extensions/nodes"
+	"github.com/rancher/rancher/tests/framework/extensions/workloads/pods"
+	"github.com/rancher/rancher/tests/framework/pkg/config"
+	"github.com/rancher/rancher/tests/framework/pkg/session"
+	"github.com/rancher/rancher/tests/v2/validation/provisioning"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+const (
+	defaultNS                = "fleet-default"
+	etcdLabel                = "rke.cattle.io/etcd-role"
+	clusterLabel             = "cluster.x-k8s.io/cluster-name"
+	machineSteveResourceType = "cluster.x-k8s.io.machine"
```

Comment:
If we're gonna use the `defaultNS` and labels in multiple places, maybe we can move them to `config.go` so that we can export them. For the machine steve resource. I don't have a strong opinion but we can have even a package in extensions, we should have an example in `extensions/secrets`.

Reply:
@caliskanugur, these labels are specific to etcd and i don't think they'll be used frequently so i kept them in the etcd_node_delete_test.go file only. For `defaultNS` i removed it and exported the existing `namespace` variable in the config.go file.